src/System.Runtime.WindowsRuntime/pkg/System.Runtime.WindowsRuntime.pkgproj

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <PropertyGroup>
+    <SkipValidatePackage>true</SkipValidatePackage>
```

Comment:
Do we need these properties?

Reply:
For <HarvestStablePackage>false</HarvestStablePackage>, it is required since old version doesn't exists

Cannot harvest package 'System.Runtime.WindowsRuntime' version '4.3.0' because D:\git\corefx\packages/System.Runtime.WindowsRuntime\4.3.0 does not exist.  Harvesting is needed to redistribute assets and ensure compatiblity with the previous release.

Reply:
We can fix the harvest issue by actually harvesting the older package. Whether or not we need to do that will be decided based on my other question of whether or not the package needs the older assets that we aren't building live any longer. Please compare the older package with the new one you are producing to see if there are missing assets.

Reply:
For 
```
<SkipValidatePackage>true</SkipValidatePackage>
```

As far as I understand, If we don't add this property, We need to add InboxFrameworks, otherwise, ValidatePackage task will fail

> D:\git\corefx\Tools\Packaging.targets(1131,5): error MSB4018: System.ArgumentNullException: Value cannot be null. [D:\git\corefx\src\System.Runtime.WindowsRuntime\pkg\System.Runtime.WindowsRuntime.pkgproj]
D:\git\corefx\Tools\Packaging.targets(1131,5): error MSB4018: Parameter name: v1 [D:\git\corefx\src\System.Runtime.WindowsRuntime\pkg\System.Runtime.WindowsRuntime.pkgproj]
D:\git\corefx\Tools\Packaging.targets(1131,5): error MSB4018:    at System.Version.op_LessThanOrEqual(Version v1, Version v2) [D:\git\corefx\src\System.Runtime.WindowsRuntime\pkg\System.Runtime.WindowsRuntime.pkgproj]
D:\git\corefx\Tools\Packaging.targets(1131,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.Packaging.InboxFrameworks.IsInbox(NuGetFramework framework, Version assemblyVersion, Boolean permitRevisions) [D:\git\corefx\src\System.Runtime.WindowsRuntime\pkg\System.Runtime.WindowsRuntime.pkgproj]
D:\git\corefx\Tools\Packaging.targets(1131,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.Packaging.ValidatePackage.LoadSupport() [D:\git\corefx\src\System.Runtime.WindowsRuntime\pkg\System.Runtime.WindowsRuntime.pkgproj]
D:\git\corefx\Tools\Packaging.targets(1131,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.Packaging.ValidatePackage.Execute() [D:\git\corefx\src\System.Runtime.WindowsRuntime\pkg\System.Runtime.WindowsRuntime.pkgproj]

Reply:
>Do you mean the old package in https://www.nuget.org/packages/System.Runtime.WindowsRuntime/?

Yes the latest one from there.

>As far as I understand, If we don't add this property, We need to add InboxFrameworks, otherwise, ValidatePackage task will fail

Perhaps we are working around a bug in our package validation in which case it might be OK to keep and add a comment for.

Reply:
Why the new nuget package should contains same asset as old nuget package?
I took a look the old nuget package: it contains lib/ref/runtime three folders, but for this onew nuget package it can only contains one folder: ref.

Reply:
The old nuget package's ref folder contains netcore50/netstand1.0/netstand2.1. For this new nuget package, do we also need support these?

The old nuget package's runtime folder contains win8/win8-aot.

Reply:
The nuget packages need to remain backwards compatible so people can always update to the latest version and remain working. So it sounds like we are going to need to harvest some assets in order for it to continue working for older platforms.

To harvest you need to add the older package to https://github.com/dotnet/corefx/blob/master/external/harvestPackages/harvestPackages.props#L4, and then we need to add HarvestIncludePaths similar to https://github.com/dotnet/corefx/blob/master/src/Microsoft.CSharp/pkg/Microsoft.CSharp.pkgproj#L9 for all the assets we want to harvest.

Reply:
You don't necessarily need the exact same structure, but you do need to make sure the package supports all the same frameworks.

> Cannot harvest package 'System.Runtime.WindowsRuntime' version '4.3.0' because D:\git\corefx\packages/System.Runtime.WindowsRuntime\4.3.0 does not exist. Harvesting is needed to redistribute assets and ensure compatiblity with the previous release.

Add a reference to https://github.com/dotnet/corefx/blob/master/external/harvestPackages/harvestPackages.props.  This ensures we download the package and will test its supported frameworks.

If you want to copy assets from the old package use HarvestIncludePaths.

> Perhaps we are working around a bug in our package validation in which case it might be OK to keep and add a comment for.

The error is bad, but its just because we didn't check for it to get a proper error message.

Reply:
Added these nuget package and remove runtime for netcoreapp2.1-- I think we only need reference assembly for netcoreapp2.1.

Reply:
The old package contains System.Runtime.WindowsRuntime.xml resource related file, the ref/netcoreapp2.1 doesn't contains this resource file, is it expected?

Reply:
> I think we only need reference assembly for netcoreapp2.1.

If you do that you'll need to have a placeholder to make sure you obscure a less-specific implementation (eg: like runtimes/win8/lib/netstandard1.3)

Reply:
so bring the runtime/netcoreapp2.1 back is better than add a placeholder?

Reply:
You just need to think about it and make sure you implement the behavior you want.  If you leave out the asset you better do something to make sure some other asset that's considered "compatible" doesn't apply...  The package report can help you reason about this, you'll find it in the output.

Reply:
I can add the following to create a placeholder to avoid other compatible apply.
```
<InboxOnTargetFramework Include="netcoreapp2.1" />
```

also after enabling package verification,  package reports give a lot errors, such as 

>
error : Framework Windows,Version=v8.0 should support System.Runtime.WindowsRuntime inbox but was missing a placeholder for run-time.  You may need to add <InboxOnTargetFramework Include="win8" /> to your project.
 >
error: Framework Windows,Version=v8.0 should support System.Runtime.WindowsRuntime inbox but contained a reference assemblies: ref/netstandard1.0/System.Runtime.WindowsRuntime.dll.  You may need to add <InboxOnTargetFramework Include="win8" /> to your project.
>
error : System.Runtime.WindowsRuntime should be supported on .NETCoreApp,Version=v2.1/win7-x86 but has no runtime assets.